pkg/client/rules.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/grafana/cortex-tools/pkg/rules/rwrulefmt"
 )
 
+const (
+	rulerAPIPath  = "/api/v1/rules"
+	legacyAPIPath = "/api/prom/rules"
+)
```

Comment:
Move this snippet to the same file where they are used.

Reply:
The snippet has moved to pkg/client/client.go